include/mbedtls/pkcs11.h

```diff
@@ -56,47 +56,81 @@ typedef struct mbedtls_pkcs11_context
         int len;
 } mbedtls_pkcs11_context;
 
+/*
+ * NOTE: It is not necessary to enclose the definitions with
+ * MBEDTLS_DEPRECATED_REMOVED because we already have a check for the full
+ * MBEDTLS_PKCS11_C feature that makes compilation fail even before we get
+ * here.
```

Comment:
I don't follow this argument. What if a 3rd party .c file includes this header but not `check_config.h`?